pkg/provision/ubuntu.go

```diff
@@ -155,6 +155,11 @@ WantedBy=multi-user.target
 		return nil, err
 	}
 
+	// because in kic base image we pre-install docker it already has a service file. we need to daemon-reload for the new systemd file
+	if err := p.Service("", serviceaction.DaemonReload); err != nil {
```

Comment:
Can we also do this in buildroot.go?

Otherwise we'll run into a similar problem when we update Docker configs to be different than a previously run.

Reply:
ok :)